docs/source/glossary.rst

```diff
@@ -19,21 +19,41 @@ This glossary defines general Sawtooth terms and concepts.
     blockchain is linked by a cryptographic hash to the previous block.
 
   Consensus
-    Process of building agreement among a group of mutually distrusting
-    participants (other nodes on a Sawtooth network). Sawtooth allows
-    different types of consensus on the same blockchain.
-    See also :term:`Dynamic consensus`.
+    Process of reaching agreement among a group of "mutually distrusting"
```

Comment:
"Mutually distrusting" is technically too specific. Non-byzantine tolerant consensus requires trust.

I would generalize this to say "The process of reaching agreement amongst a group of participants..."